features/vcr_cassettes/Search_-_redesign/Search_with_video_results.yml

```diff
@@ -0,0 +1,387 @@
+---
+http_interactions:
+- request:
```

Comment:
@ddzz Sorry, nit-picky, but the first 100ish lines in this recording are unnecessary noise likely recorded as you iteratively worked on this test.  Can you delete this recording then rerun the test to get a clean initial recording for this?  (I just did this here locally without capturing these extraneous dd calls and error messages.)

That said, I'm also going to go ahead an "approve" because this is in no way a blocker, just a preference.  If something is up with your environment that a delete -> rerecord doesn't remedy this, there's no reason to spend more time on it.  We can troubleshoot another day.

Reply:
I did do a delete and re-record to remedy a very large cassette. I just tried again and it didn't affect the cassette size, so I don't think it's worth extra time perfecting this. It's a small cassette anyway.